ChangeLog.txt

```diff
@@ -1,6 +1,61 @@
 2023-XX-XX v0.1.5
 -----------
 
+Mechanics:
+- Minorities:
+   - Faith Converison (by event/decision/interface): a large minority of your old faith will still exist in your capital if you convert
+   - Culture creation removes parent culture minorities in counties of the new culture
+   - If a hybrid or diverged culture is within the cooldown period, the parent cultures are subsumed instead of becoming a minority when the county culture changes
+   - Conversion (by any means) now will keep a large minority of the old faith in your capital county
+
+Defines & Game Rules:
+- Added default game rule slowing hybrid and divergence
+- Reduced a number of hard-coded defines, slowing the growth of cultural acceptance
+
+Cultures:
+- Add ability to have more than one parent culture for historical cultures (enabling hybridization locks)
+- Updated custom culture name triggers so they should appear more often in games
+- Culture Tweaks:
+   - Sardinian starts with Proto-Sardinian instead of Sardinian Vulgar in 476
+   - Subei (as in, Iberian Suebi) replaces Agrarian with Strong Believers
+   - Lusitanian is a parent culture of Romano-Hispanic (to prevent hybridization)
+- Tradition Changes:
+   - Visigothic replaces Ruling Caste with Isolationist
+   - Ostrogothic replaces Formation Fighting Experts with Isolationist
+   - Moseogothic replaces Tribal Unity with Isolationist
```

Comment:
I'm not entirely convinced by these tradition changes. I agree with reducing hybridisation, but as we discussed I think it is a general issue as opposed to specific for these three cultures

Reply:
Then I'll revert these changes; they were made with the idea that Ruling Caste has the same effect on the hybridization event, but that would _also_ make it unlikely for the Visigoths to become a Romance Heritage culture.